libparsec/crates/platform_device_loader/src/lib.rs

```diff
@@ -26,9 +26,9 @@ pub const ARGON2ID_DEFAULT_PARALLELISM: u32 = 1;
 
 pub(crate) const DEVICE_FILE_EXT: &str = "keys";
 
-pub const PARSEC_CONFIG_DIR: &str = "PARSEC_CONFIG_DIR";
-pub const PARSEC_DATA_DIR: &str = "PARSEC_DATA_DIR";
-pub const PARSEC_HOME_DIR: &str = "PARSEC_HOME_DIR";
+pub const PARSEC_BASE_CONFIG_DIR: &str = "PARSEC_BASE_CONFIG_DIR";
+pub const PARSEC_BASE_DATA_DIR: &str = "PARSEC_BASE_DATA_DIR";
+pub const PARSEC_BASE_HOME_DIR: &str = "PARSEC_BASE_HOME_DIR";
```

Comment:
What is the rationale for this change? (it is not described in the PR description)

Reply:
The description was put in the 3 commit, but not in the PR description

## The difference between `BASE_DIR` and `DIR`

- `BASE_DIR` as the name suggest, it's used as a base where it's folder is used to add sub-folder (`{BASE_DIR}/foo/bar`).
- `DIR`, it's used as is (`{DIR}/foo.txt`).

## Why did I made this change?

The mean reason was to be consistent, 
We had functions like `get_default_config_dir` that used `PARSEC_CONFIG_DIR` as a base value where a subpath was added to it (`{CONFIG_DIR}/parsec3/libparsec`).

That conflict with the CLI where the value will be used as is without additional transformation (i.e.: adding a subpath).

So that the reason I choose to rename those variable used in `platform-device-loader` to include `_BASE` since we add a subpath to it. 

But we could also rework the functions where the env var is used as is without adding a subpath (to copy the behavior of the CLI).